testing/src/cypress/page-objects/Institution-objects/ManageDesignationsObject.ts

```diff
@@ -6,4 +8,124 @@ export default class ManageDesignationsObject {
   designationAgreementsText() {
     return cy.contains("Designation agreements");
   }
+
+  viewDesignationButton() {
+    return cy.contains("View");
```

Comment:
Are we searching the element based on the content?
Any reason why we are not using `data-cy` for these ones?

Reply:
I am only trying to use content only when unique - in such way that any change (content/CTA) in that is essentially indicative of a `bug`. But we can discuss

Reply:
I see your point. I would say that the idea is to make tests resilient to fail in the case of a content change.
I would recommend doing the select always using the `data-cy`. If the idea is to ensure the content, we can select using the `data-cy` and then test the content. In this way, only the test asserting the content would fail. For instance, if the selector is used across 15 differents tests, 14 would still pass and only the content one would fail.

Reply:
IMO I would suggest the same as we may have a huge content revamp coming down the road which may lead to lot of test case fixing.